amr-wind/physics/multiphase/MultiPhase.cpp

```diff
@@ -296,6 +296,54 @@ void MultiPhase::set_nph_density()
         m_density.num_comp(), 1);
 }
 
+// Using phase densities to convert advected VOF arrays to advected density
```

Comment:
Yes, it would make sense to be in the vof struct since it is a post-solve operation. But the other two post-solve operations belong to the MultiPhase class, so I was thinking it would match those.

Reply:
This location also helps provide access to the phase densities.